README.md

```diff
@@ -142,3 +142,121 @@ Options:
+```
+
+Type your password in twice and you will be given a fresh and newly generated
+account.
+
+```zsh
+Please note that password is NOT RECOVERABLE.
+Type password: 
+Repeat password: 
+0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2
+```
+
+Now since we will only be using openethereum for its keystore, we can start
+the service using the following flags to disable networking and syncing.
+
+```zsh
+openethereum --chain goerli --max-peers 0 --ws-apis all
+```
+
+### Using injection-tool
+
+Now open a new terminal, we'll be using most of the Ethereum functionality
+in injection-tool now to manage our own deployment of a `FrozenToken` and
+`Claims` contract.
+
+> Note: The instruction below assume you have the injection-tool source code
+> locally and that you're using `ts-node` to run the commands. An alternative
+> is to install the injection-tool from NPM with `npm i @w3f/injection-tool -g`.
+> After installing you would replace `ts-node src/index` in the commands below
+> with the `injection-tool` command.
+
+```zsh
+ts-node src/index eth:frozenToken-deploy --nonce 0 --output frozen.raw.tx --owner 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --from 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --password <your_password>
+```
+
+Now we can submit this to the network using the broadcast command.
+
+```zsh
+ts-node src/index eth:broadcast --csv frozen.raw.tx --providerUrl wss://goerli.infura.io/ws/v3/7121204aac9a45dcb9c2cc825fb85159
+```
+
+The transaction will be broadcast to the node and the script will wait until
+it's mined and a receipt is received. It will then print this receipt to a 
+`receipts` file and close the process.
+
+In the receipts file ctrl-f and look for `transactionHash` copy the hash that's
+given there and enter it into a block explorer like Etherscan to see the details. If everything went right, you should see the transaction succeeded.
+
+Now we will deploy the claims contract. From the deployment transaction before
+we can grab the address for our token (the "dot indicator") from Etherscan.
+Use the address for the `--dotIndicator` option below.
+
+```zsh
+ts-node src/index eth:claims-deploy --nonce 1 --output claims.raw.tx --dotIndicator 0x10068eBE0665BB6d7a58deBB0C1c262849613505 --owner 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --from 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --password <your_password>
+```
+
+Use the broadcast command like before:
+
+```zsh
+ts-node src/index eth:broadcast --csv claims.raw.tx --providerUrl wss://goerli.infura.io/ws/v3/7121204aac9a45dcb9c2cc825fb85159
+```
+
+#### Allocations
+
+```zsh
+ts-node src/index eth:dot-allocations --nonce 2 --output allocations.raw.tx --csv allocations.csv --frozenToken 0x10068eBE0665BB6d7a58deBB0C1c262849613505 --from 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --password <your_password>
+```
+
+and 
+
+```zsh
+ts-node src/index eth:broadcast --csv allocations.raw.tx --providerUrl wss://goerli.infura.io/ws/v3/7121204aac9a45dcb9c2cc825fb85159
+```
```

Comment:
this is invalid. you will get an error when submitting the tx since the `transfer` function only allows the contract owner to call, and you are the only the owner, which means that they have to deploy the frozenToken contract by themself.

Reply:
The first two steps is deploying the frozen token and the claims contract.